src/ui/server/middleware/integrity/index.test.ts

```diff
@@ -0,0 +1,59 @@
+import { mockNext, mockReq, mockRes } from '../../test-mocks';
+import { Request, Response } from '../../../types';
+import { integrity } from '.';
+import { INTEGRITY } from '../../constants';
+
+const { GOVUK, FORM, COOKIES, GA, MOJ, ACCESSIBILITY } = INTEGRITY;
+const req: Request = mockReq();
+const res: Response = mockRes();
+const next = mockNext;
+
+describe('sub resource integrity (SRI) middleware unit test cases', () => {
```

Comment:
For all new middleware can we match the other test descriptions? Just the directory name

e.g

```js
describe('middleware/cookies-consent', () => {
```

I don't think we need to mention they're unit tests in the describe block